wiki/Gameplay/Daily_challenge/de.md

```diff
@@ -0,0 +1,60 @@
+![](img/daily-challenge-menu.png "Screenshot der täglichen Herausforderung am 7. November 2024.")
+
+Beim Öffnen der täglichen Herausforderung wird ein Intro angezeigt, das die abzuschließende Beatmap sowie die vorgegebenen Mods präsentiert.
+
+Nachdem das Intro endet können Nutzer die Daten der Herausforderung für den jeweiligen Tag einsehen. Die linke Seite stellt verschiedene Informationen zu den Punkteständen bereit, wie beispielsweise die Gesamtzahl der abgeschlossenen Durchläufe und die kumulierte Gesamtpunktzahl. In der Mitte zeigt eine Rangliste die Highscores von Spielern an. Nutzer können sich im Chat auf der rechte Seite über die tägliche Herausforderung austauschen.
+
+## Schwierigkeitsbereich für Beatmaps
+
+![](img/Beatmap-selection-criteria.png "Beatmaps müssen an den Wochentagen eine bestimmte Sternenanzahl haben")
+
+## Meilensteine
+
+Spielserien werden auf dem Nutzerprofil abhängig von den erreichten Meilensteinen mit unterschiedlichen Farben dargestellt:
+
+<!-- tier images: https://www.figma.com/design/tc79qAgJ35KQvdTO0Oj3dN/Daily-Challenge-Counter?node-id=0-1&t=xjRm9Ke0tUMtAQlh-1 -->
+
+|  | Rang | Gesamtteilnahme | Tägliche Serie | Wöchentliche Serie |
+| --: | :-: | :-: | :-: | :-: |
+| ![Lustrous tier](img/tiers/lustrous.png) | Lustrous | 1080 Tage | 360 Tage | 53 Wochen |
+| ![Radiant tier](img/tiers/radiant.png) | Radiant | 720 Tage | 240 Tage | 36 Wochen |
+| ![Rhodium tier](img/tiers/rhodium.png) | Rhodium | 360 Tage | 120 Tage | 19 Wochen |
+| ![Platinum tier](img/tiers/platinum.png) | Platinum | 180 Tage | 60 Tage | 10 Wochen |
+| ![Gold tier](img/tiers/gold.png) | Gold | 90 Tage | 30 Tage | 6 Wochen |
+| ![Silver tier](img/tiers/silver.png) | Silver | 30 Tage | 10 Tage | 3 Wochen |
+| ![Bronze tier](img/tiers/bronze.png) | Bronze | 15 Tage | 5 Tage | 2 Wochen |
+| ![Iron tier](img/tiers/iron.png) | Iron | weniger als 15 Tage | weniger als 5 Tage | weniger als 2 Wochen |
+
+## Mitwirkende
+
+Das Projekt wird von ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) geleitet. Die folgenden Community-Mitwirkenden sind für die Auswahl der Beatmaps verantwortlich:
+
+- ::{ flag=SE }:: [bite you death](https://osu.ppy.sh/users/6398464)
+- ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251)
+- ::{ flag=BR }:: [Dada](https://osu.ppy.sh/users/9119507)
+- ::{ flag=KR }:: [momoyo](https://osu.ppy.sh/users/12469536)
+- ::{ flag=MX }:: [Riot](https://osu.ppy.sh/users/4256461)
+- ::{ flag=US }:: [TheMagicAnimals](https://osu.ppy.sh/users/17274052)
+
+## Trivia
+
+::: Infobox
+![](img/yt-comment.jpg "YouTube-Kommentar von waxxx14, der ein \"cup of the day\"-Format vorgeschlagen hatte")
+:::
+
+- Die Idee für die tägliche Herausforderung kam von waxxx14, der ein "cup of the day"-Format wie in [TrackMania](https://de.wikipedia.org/wiki/TrackMania) vorgeschlagen hatte. Der Beitrag wurde unter dem lazer-Entwicklungsvideo ["deciding what to do with lazer"](https://www.youtube.com/watch?v=xUSxEjQQ1UI) veröffentlicht.
```

Comment:
Plusquamperfekt here is a bit unusual:

```suggestion
- Die Idee für die tägliche Herausforderung kam von waxxx14, der ein "cup of the day"-Format wie in [TrackMania](https://de.wikipedia.org/wiki/TrackMania) vorgeschlagen hat. Der Beitrag wurde unter dem lazer-Entwicklungsvideo ["deciding what to do with lazer"](https://www.youtube.com/watch?v=xUSxEjQQ1UI) veröffentlicht.
```

"vorschlug" is also an option but sounds fairly formal (it's rare to use the Präteritum in normal speak I think). or maybe another word altogether?